### PR TITLE
Feature/azavea.opentripplanner

### PR DESCRIPTION
**Pull request by CloudNiner**
Add VM for Open Trip Planner, using azavea's fork, v0.13.0

### CODE REVIEW

File: deployment/ansible/roles/nih-wayfinding.otp-data/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+## Defined in azavea.opentripplanner
+#otp_bin_dir: /opt/opentripplanner
+#otp_data_dir: /var/otp
+#otp_jarfile: "./target/otp-0.13.0.jar"
+
```
Comment (flibbertigibbet) [inline deployment/ansible/roles/nih-wayfinding.otp-data/defaults/main.yml:2]: I suppose the intent is to take this commented stuff out once the role is modified?

File: deployment/ansible/group_vars/all [hunk 2]
```diff
@@ -19,4 +19,8 @@ postgresql_support_libpq_version: "9.3.*"
 postgresql_support_repository_channel: "9.3"
 postgresql_version: "9.3"
 
+otp_repo: "https://github.com/azavea/OpenTripPlanner.git"
+otp_version: "77072a5b548e1777b23127ebedfcf08f2461c171"
+otp_jarfile: "./target/otp-0.13.0.jar"
```
Comment (flibbertigibbet) [inline deployment/ansible/group_vars/all:24]: Could also just use a wildcard instead, like `otp-*.jar`.  I think that's what the `otp` script does.

**Comment by flibbertigibbet** [comment]
+1

It would be nice to just pull from the head of the develop branch, instead of using a commit; I guess we'll modify that later.

File: deployment/ansible/group_vars/all (hunk 2 above)
Comment (CloudNiner) [inline deployment/ansible/group_vars/all:24]: Yeah, I initially tried that, and it grumped hard. I didn't look into it too much though. Could be a wildcard unescaping at the wrong time.

**Comment by CloudNiner** [comment]
Yeah, agreed. Version allows the string 'HEAD' or a branch name. I'll update it to 'develop' now.

File: deployment/ansible/roles/nih-wayfinding.otp-data/defaults/main.yml (hunk 1 above)
Comment (CloudNiner) [inline deployment/ansible/roles/nih-wayfinding.otp-data/defaults/main.yml:2]: Nope, this is leftover stuff that can be removed.

**Pull request closed without merging**